p2p/discover: flaky TestUDPv5_findnodeHandling

#### System information

Geth version: master (commit c3467dd8b)
OS & Version: Linux (GitHub Actions CI), macOS (local)

#### Expected behaviour

`TestUDPv5_findnodeHandling` should pass reliably.

#### Actual behaviour

The test intermittently fails with:

```
--- FAIL: TestUDPv5_findnodeHandling (0.01s)
    panic.go:694: 1 unmatched UDP packets in queue
```

This has been observed in CI (e.g. run [#23626948218](https://github.com/ethereum/go-ethereum/actions/runs/23626948218)) and can be reproduced locally with:

```
go test -race -run TestUDPv5_findnodeHandling -count=50 ./p2p/discover/
```

#### Root cause

`TestUDPv5_findnodeHandling` inserts nodes into the routing table via `fillTable`. The table's revalidation loop (`tableRevalidation.run`) may schedule PING packets for these nodes, which are dispatched through the transport and written to the test's `dgramPipe`. The test's `close()` method asserts that the pipe queue is empty, but these background PING packets can arrive after the test's last `expectNodes` call completes, leaving unmatched packets in the queue.

The default `PingInterval` is 3 seconds, and under the race detector the test runs slowly enough for revalidation to fire.

The v5 test harness (`udpV5Test`) does not use a simulated clock, so real-time revalidation timers can fire during the test.

--- a/p2p/discover/v5_udp_test.go
+++ b/p2p/discover/v5_udp_test.go
@@ -937,6 +937,7 @@ func newUDPV5Test(t *testing.T) *udpV5Test {
 		PrivateKey:   test.localkey,
 		Log:          testlog.Logger(t, log.LvlTrace),
 		ValidSchemes: enode.ValidSchemesForTesting,
+		PingInterval: 1000 * time.Hour,
 	})
 	test.udp.codec = &testCodec{test: test, id: ln.ID()}
 	test.table = test.udp.tab

